tools/pot/tests/test_model_params.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (C) 2020-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+import numpy as np
+import pytest
+from addict import Dict
+
+from openvino.tools.pot import DataLoader, IEEngine, create_pipeline, load_model
+from openvino.tools.pot.graph.model_utils import get_nodes_by_type, get_node_by_name
+from openvino.tools.pot.graph.node_utils import get_bias_for_node, get_node_value
+from openvino.tools.pot.graph.special_operations import OPERATIONS_WITH_BIAS
+
+from .utils.config import merge_configs
+from .utils.data_helper import dump_intermediate_data, get_ref_stats
+
+
+EPS = 1e-6
+
+
+class SyntheticDataLoader(DataLoader):
+    def __init__(self, config):
+        self._samples = self._generate_synthetic_data(config)
+
+    def __len__(self):
+        return len(self._samples)
+
+    def __getitem__(self, index):
+        return self._samples[index], index
+
+    def _generate_synthetic_data(self, config):
+        samples = []
+        np.random.seed(config.seed)
+        for _ in range(config.num_samples):
+            sample = np.random.normal(size=config.input_shape)
+            samples.append(sample)
+
+        return samples
```

Comment:
It would be great to reuse `SyntheticDataLoader` with different input ranges in `test_scales.py`.

https://github.com/openvinotoolkit/openvino/blob/ab9d2bf8c0736c2078d2eff6a8cf78661c713668/tools/pot/tests/test_scales.py#L303-L316

Reply:
Missed it, thanks. I will update the code soon.